language/Portuguese_Brazil/strings.po

```diff
@@ -392,23 +395,23 @@ msgstr "Cores"
 
 #: src/frontend/mame/ui/custui.cpp:157 src/frontend/mame/ui/dirmenu.cpp:38
 msgid "Language"
-msgstr "Linguagem"
+msgstr "Idioma"
 
 #: src/frontend/mame/ui/custui.cpp:161
 msgid "Show side panels"
 msgstr "Mostrar painéis laterais"
 
 #: src/frontend/mame/ui/custui.cpp:173
 msgid "Custom UI Settings"
-msgstr "Configurações Customizadas do UI"
+msgstr "Configurações Personalizadas da IU"
 
 #: src/frontend/mame/ui/custui.cpp:223
 msgid "default"
 msgstr "padrão"
 
 #: src/frontend/mame/ui/custui.cpp:331
 msgid "UI Font"
-msgstr "Fonte do UI"
+msgstr "Fonte da IU"
```

Comment:
Similar issue here with the uncommon **IU** usage.

You know... in the early 80s people in Brazil used to translate **CPU** to **UCP** (Unidade Central de Processamento). But nobody does that anymore! Tech-lingo broadly adopts acronyms in its original english form nowadays (such as CPU and UI or even GUI).

Reply:
here I'd suggest saying something like "Tipografia da Interface de Usuário". We may consider using just "Interface" in places where "Interface de Usuário" results in too long strings.

Reply:
Even "Fonte Tipográfica" (not mentioning the UI at all) would work well in this case, though. And could be a good / effective short string to be used here.